languages/wikipron/src/codes.py

```diff
@@ -0,0 +1,174 @@
+"""
+This is a tool for scraping all languages with over 100 entries from:
+https://en.wiktionary.org/wiki/Category:Terms_with_IPA_pronunciation_by_language
+For each language it grabs the language name and
+language code (likely ISO 639-1) that Wiktionary uses.
+It compares that code  with those in "./iso-639-3_20190408.tsv"
+in order to grab the appropriate ISO 639-2 or ISO 639-3 code and language name.
+A dictionary containing this data is created and
+converted to a JSON file (languages.json).
+Casefolding settings for languages already in languages.json
+are transferred to the new languages dictionary being created.
+
+New languages that are added through this process
+and outputted to languages.json require further processing
+before being imported into scrape_and_write.py.
+Casefolding still needs to be specified.
+Dialect information and other config options need
+to be added manually as well.
+"""
+
+import requests
+import requests_html
+import re
+import csv
+import json
+
+
+def cat_info(cat_title):
+    cat_info_params = {
+        "action": "query",
+        "format": "json",
+        "titles": cat_title,
+        "prop": "categoryinfo"
+    }
+    data = requests.get(
+        "https://en.wiktionary.org/w/api.php?",
+        params=cat_info_params
+    ).json()
+    pages = data["query"]["pages"]
+
+    for k, v in pages.items():
+        num_of_pages = v["categoryinfo"]["pages"]
+        if num_of_pages >= 100:
+            isolate_language_category = re.search(
+                r"(\s+)terms(\s+)",
+                v["title"]
+            )
+            yield (
+                v["title"][0:isolate_language_category.start()],
+                num_of_pages
+            )
+        else:
+            continue
```

Comment:
This `else: continue` bit seems unnecessary, since the code will `continue` anyway if the if-condition is false?